test/e2e/csv_e2e_test.go

```diff
@@ -306,7 +320,7 @@ var _ = Describe("ClusterServiceVersion", func() {
 				}
 				Expect(ctx.Ctx().Client().Create(context.Background(), &ns)).To(Succeed())
 
-				og := operatorsv1.OperatorGroup{
+				og = operatorsv1.OperatorGroup{
```

Comment:
Any idea on whether we may need to also auto-generate the "test-namespace-2" that gets created [above this](https://github.com/operator-framework/operator-lifecycle-manager/pull/2541/files#diff-e94bc9e5ebd2b6db20f851288184630432208a5a772cf0515935707ad3660799R318) variable declaration in the BeforeEach clause to avoid any potential flakes down the line around attempting to create a namespace/CRD/etc. that already exists (e.g. it wasn't properly cleaned up for whatever reason)? 

If I'm reading this code right, we'll have the top-level BeforeEach create a testing namespace (that gets autogenerated through the `genName(...)` call), and then we'll create a `test-namespace-2` namespace to test collision for each test spec defined in this nested when container.

Reply:
Yes, it's better to be with the generated name.  I'll change it.  Thanks!